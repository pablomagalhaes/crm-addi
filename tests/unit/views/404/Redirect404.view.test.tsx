import React from 'react';
import { shallow } from 'enzyme';

import Redirect404View from '@/views/404/Redirect404.view';

describe('<Redirect404 />', () => {
  it('should match snapchat', () => {
    const wrapper = shallow(<Redirect404View />);
    expect(wrapper).toMatchSnapshot();
  });
});
