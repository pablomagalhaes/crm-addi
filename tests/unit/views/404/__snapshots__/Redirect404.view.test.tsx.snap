// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Redirect404 /> should match snapchat 1`] = `
<Redirect
  to="/"
/>
`;
